javascript-big-practice/src/pages/main.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
html and css should be empty or just skeleton for now. This MR is only for structure. Just do it step by step